lib/endpoints/class-wp-rest-taxonomies-controller.php

```diff
@@ -107,6 +107,7 @@ public function prepare_item_for_response( $taxonomy, $request ) {
 		$data = array(
 			'name'         => $taxonomy->label,
 			'slug'         => $taxonomy->name,
+			'capabilities' => $taxonomy->cap,
```

Comment:
Does it make sense to white-list the capabilities? I'm wondering if in other versions of WordPress or plugins making modifications with new capabilities that we want consistent data coming from the API.